### PR TITLE
Update handler.py.jinja (fixes #1)

### LINKED ISSUE #1: A few changes in handler.py template
Hi there, this template is very helpful! Looking forward to building a custom handler!

I noticed a few discrepencies in my generated `handler.py`, I believe there are some changes needed in [`handler.py.jinja`](https://github.com/mkdocstrings/handler-template/blob/1978f7a778fbd04c75bb2646317eafcf85cf3b9d/project/src/mkdocstrings_handlers/%7B%7Bpython_package_import_name%7D%7D/handler.py.jinja)

- [x] The comment on Line 1 still has `Python` hardcoded, I believe this should be `{{ language|title }}`

   https://github.com/mkdocstrings/handler-template/blob/1978f7a778fbd04c75bb2646317eafcf85cf3b9d/project/src/mkdocstrings_handlers/%7B%7Bpython_package_import_name%7D%7D/handler.py.jinja#L1

- [x] The class name generated will fail if the user enters spaces in the language name. I would have PRed a fix for both of these, but I wasn't sure what Jinja filter to use to transform a name like `"Language Name"` to `LanguageName`

   https://github.com/mkdocstrings/handler-template/blob/1978f7a778fbd04c75bb2646317eafcf85cf3b9d/project/src/mkdocstrings_handlers/%7B%7Bpython_package_import_name%7D%7D/handler.py.jinja#L16

### PR DESCRIPTION
**Pull request by weaversam8**
Fixes #1.

### CODE REVIEW

File: project/src/mkdocstrings_handlers/{{python_package_import_name}}/handler.py.jinja [hunk 1]
```diff
@@ -13,8 +13,8 @@ from mkdocstrings.loggers import get_logger
 logger = get_logger(__name__)
 
 
-class {{ language|title }}Handler(BaseHandler):
-    """The {{ language|title }} handler class.
+class {{ language.title().replace(" ", "") }}Handler(BaseHandler):
+    """The {{ llanguage.title().replace(" ", "") }} handler class.
```
Comment (pawamoy) [inline project/src/mkdocstrings_handlers/{{python_package_import_name}}/handler.py.jinja]: Extraneous `l`!

**Review by pawamoy — CHANGES_REQUESTED** [review]
Nice, thanks!

File: project/src/mkdocstrings_handlers/{{python_package_import_name}}/handler.py.jinja (hunk 1 above)
Comment (weaversam8) [reply project/src/mkdocstrings_handlers/{{python_package_import_name}}/handler.py.jinja]: Good catch, thanks!

**Review by pawamoy — APPROVED** [review]
Looking good, thanks a lot!

**Comment by pawamoy** [comment]
I'm gonna fix CI for the template, don't worry about it.

**Pull request merged**